### PR TITLE
doc: adds attribution and updates changelog

### PR DESCRIPTION
**Pull request by adam-hanna**
Please go the the `Preview` tab and select the appropriate sub-template:

* [Production code](?expand=1&template=production.md) - for types `fix`, `feat`, and `refactor`.
* [Docs](?expand=1&template=docs.md) - for documentation changes.
* [Others](?expand=1&template=others.md) - for changes that do not affect production code.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,20 @@
...
+- [Forked cosmos-sdk/x/gov](https://github.com/atomone-hub/atomone/commit/ca0724f036f077ffd3b2efc2a43db2ed98ad885e): Forked x/gov
+- [#5](https://github.com/atomone-hub/atomone/pull/5): Disabled validator vote inheritance
+- [#7](https://github.com/atomone-hub/atomone/pull/7): Removed usage of v1.ValidatorGovInfo
+- [#8](https://github.com/atomone-hub/atomone/pull/8): Added x/gov/types
+- [#9](https://github.com/atomone-hub/atomone/pull/9): Remove No With Veto, and set default pass threshold to 2/3 and quorum to 25%
+- [#10](https://github.com/atomone-hub/atomone/pull/10): Changed min voting period to 3 weeks
+- [#11](https://github.com/atomone-hub/atomone/pull/11): Added constitution amendment and law proposals with specific quorum and pass thresholds
+- [#13](https://github.com/atomone-hub/atomone/pull/12): Added late quorum voting period extension
+- [#13](https://github.com/atomone-hub/atomone/pull/13): Added a minimum amount per deposit
+- [#15](https://github.com/atomone-hub/atomone/pull/15): Added constitution to x/gov
```
Comment (tbruyelle) [inline CHANGELOG.md:27]: Nice, it's more complete than my attempt in #18. One tiny thing is about the format, usually it's the text first, then the link to the PR. I think we should stick to this.

Comment (giunatale) [reply CHANGELOG.md:27]: yeah I think aside from @tbruyelle comment this looks pretty good! TY

Comment (tbruyelle) [reply CHANGELOG.md]: @giunatale I reformatted the entries, should be ready to merge now.

**Comment by giunatale** [comment]
@tbruyelle let's leave this open so we keep updating the changelog until all PRs are merged in

**Review by giunatale — APPROVED** [review]

**Pull request merged**